From 5c6f5edd22fd3beddf0a2308031fd21a557e8513 Mon Sep 17 00:00:00 2001
From: Haibo Chen <haibo.chen@nxp.com>
Date: Fri, 13 Apr 2018 15:03:00 +0800
Subject: [PATCH 3596/5242] MLK-18026 dts: imx7ulp-evk-sd1: delete
 non-removable property for sd1 slot

commit  4815a56ef3c83dfee3a56c2e3562d1039643bba8 from
https://source.codeaurora.org/external/imx/linux-imx.git

commit a56e6e190015 ("MLK-17961 dts: imx7ulp-evk: add non-removable
property for wifi sdio") add non-removable property, sd1 slot on
base board share the same usdhc with wifi, and the sd1 slot support
card detect, so for sd1 slot, need to remove the non-removable
property.

Signed-off-by: Haibo Chen <haibo.chen@nxp.com>
Reviewed-by: Andy Duan <fuguang.duan@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7ulp-evk-sd1.dts |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/imx7ulp-evk-sd1.dts b/arch/arm/boot/dts/imx7ulp-evk-sd1.dts
index 9936183..c58e8e0 100644
--- a/arch/arm/boot/dts/imx7ulp-evk-sd1.dts
+++ b/arch/arm/boot/dts/imx7ulp-evk-sd1.dts
@@ -41,5 +41,6 @@
 	/delete-property/wifi-host;
 	/delete-property/pm-ignore-notify;
 	/delete-property/keep-power-in-suspend;
+	/delete-property/non-removable;
 	status = "okay";
 };
-- 
1.7.9.5

